src/actions/files.js

```diff
@@ -1,17 +1,22 @@
-import * as Sentry from '@sentry/react';
-import { athena as Athena, devices as Devices, raw as Raw } from '@commaai/api';
+import * as Sentry from "@sentry/react";
+import { athena as Athena, devices as Devices, raw as Raw } from "@commaai/api";
 
-import { updateDeviceOnline, fetchDeviceNetworkStatus } from '.';
-import * as Types from './types';
-import { deviceOnCellular, getDeviceFromState, deviceVersionAtLeast, asyncSleep } from '../utils';
+import { updateDeviceOnline, fetchDeviceNetworkStatus } from ".";
+import * as Types from "./types";
+import {
+  deviceOnCellular,
+  getDeviceFromState,
+  deviceVersionAtLeast,
+  asyncSleep,
+} from "../utils";
 
 const FILE_NAMES = {
-  qcameras: 'qcamera.ts',
```

Comment:
this is a large part of the diff, what's the rationale behind it? maybe we should just ignore it?

Reply:
Some of the stylistic changes are from vscode auto-formatting the code, I'm going to revert these changes.